src/test/kotlin/domaintest/LottoCustomerTest.kt

```diff
@@ -0,0 +1,32 @@
+package domaintest
+
+import domain.LottoCustomer
+import domain.model.PurchaseMoney
+import org.assertj.core.api.Java6Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+
+class LottoCustomerTest {
+
+    @Test
+    fun `사용자가 구매하려고하는 로또 전체의 개수보다 수동으로 발급받을 로또의 개수가 적은 경우`() {
+        assertDoesNotThrow {
+            LottoCustomer(PurchaseMoney(10000), 3)
+        }
+    }
+
+    @Test
+    fun `사용자가 구매하려고하는 로또 전체의 개수보다 수동으로 발급받을 로또의 개수가 많은 경우 예외가 발생`() {
+        assertThrows<IllegalArgumentException> {
+            LottoCustomer(PurchaseMoney(10000), 11)
+        }
+    }
+
+    @Test
+    fun `사용자가 구매하려고하는 자동로또의 개수를 구하기`() {
```

Comment:
```suggestion
    fun `10장 중에서 3장을 수동구매하면_자동로또는 7장이다`() {
```
테스트하고자 하는 것을 좀 더 명시적으로 드러내보면 어떨까요? (이하 관련 내용 동일)